requirements.txt

```diff
@@ -1,16 +1,16 @@
-accelerate>=1.0.0
-bitsandbytes>=0.44.1
-diffusers>=0.30.4
+accelerate>=0.34.2
+bitsandbytes>=0.43.3
```

Comment:
I think we can definitely unpin the version. It's safe.

Reply:
Sounds good!